### PR TITLE
ENDOC-303 Added authorization tutorial

### PR DESCRIPTION
**Pull request by nshaw**
@joewhite101 and @es-entando , this is a first pass just for 6.3. Once the tutorial is reviewed, I'll add it for next. Thought it would be easier to have a single version to start.

### CODE REVIEW

**Comment by nshaw** [comment]
I made another pass on the language and added the next version of the tutorial. Their content is identical so you only need to review one.

**Review by joewhite101 — APPROVED** [review]
LGTM

File: vuepress/docs/.vuepress/next.js [hunk 1]
```diff
@@ -173,6 +173,10 @@ module.exports = {
                         title: 'Update Project Data Model',
                         path: path  + 'backend-developers/update-data-model.md',
                     },
+                    {
+                        title: 'Add Authorization Checks',
```
[the remark this answers is not in the document]
Comment (nshaw) [reply vuepress/docs/.vuepress/next.js:177]: Good call. Changing to Access Controls which is more standard.

File: vuepress/docs/next/tutorials/backend-developers/add-authorization-checks.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+
+## Tutorial
+Let's start by securing the list of Conferences so only our two user roles can view the list.
+
+1. Edit `ConferenceResource.java` located in the `src/main/java/com/<ORG>/<NAME>.web.rest` directory. Modify the REST API `Conference:getAllConferences` method by adding the following annotation. 
+```
+    @PreAuthorize("hasAnyAuthority('conference-user','conference-admin')")
+    public List<Conference> getAllConferences() {
+```
+See the Spring Security documentation for more details but this restricts use of the `getConference` method to users who have been assigned either the `conference-user` or the `conference-admin` role on the Keycloak client configured for the microservice. In local testing this defaults to the `internal` client but see notes below on how that works in production.
```
[the remark this answers is not in the document]
Comment (nshaw) [reply vuepress/docs/next/tutorials/backend-developers/add-authorization-checks.md:21]: Will do

**Pull request merged**